src/test/regress/citus_tests/config.py

```diff
@@ -38,7 +38,7 @@
 
 MASTER = "master"
 # This should be updated when citus version changes
-MASTER_VERSION = "11.2"
+MASTER_VERSION = '11.3'
```

Comment:
How about keeping the syntax consistent and using double quotes like we do in other places of the same file?

```suggestion
MASTER_VERSION = "11.3"
```

Reply:
yeah will revert this